src/start/PasswordAuthView.js

```diff
@@ -90,6 +90,16 @@ class PasswordAuthView extends React.Component {
           onPress={this.validateForm}
         />
         <ErrorMsg error={error} />
+        <TouchableOpacity activeOpacity={1.5}>
```

Comment:
We have our common `Touchable`  for the project. https://github.com/zulip/zulip-mobile/blob/master/src/common/Touchable.ios.js